packages/util/src/string-formatter.spec.ts

```diff
@@ -48,6 +48,14 @@ describe('converts names', () => {
     it('for properties with numbers', () => {
       expect(camelCase('Field13Name')).toBe('field13Name');
     });
+
+    it('for properties with dot', () => {
```

Comment:
perhaps add propertiy names with 'dot' because for me it was not clear if it is about the name of the property or the value.

Reply:
This is a left over because I recently moved it from core to util. I forgot to align the names and make them more general, We can either do this as part of this PR or on a separate PR. Just let me know.

I guess something like this would be good.
```js
describe('string-formatter', () => {
  describe('camelCase', () => {
    it('transforms string containing a number', ...
    it('transforms string containing a dot', ...
    it('transforms string starting with $', ...
    etc.
  });
});
```

(I guess a separate PR would be better...)